minard/templates/shifter_information.html

```diff
@@ -0,0 +1,71 @@
+{% extends "layout.html" %}
+{% block title %} Shifter Information  {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+<div class="container">
+  <div class="row">
+    <h1 align="center">Shifter Information</h1> <br>
+    <table class="table table-hover">
+      <tr> 
+        <th> <h3> <font color='green'> {{ shifter }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ updates }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ expert }} </font> </h3> <br> </th>
+      </tr>
+    </table>
+    <h3> If you are the current shifter, please fill out the below information at the start of your shift. If you are not the current shifter, do not fill in this form. After hitting Submit, make sure the current shifter and expert information about update properly. If multiple people are on shift, the shift leader should fill this out. This information will only be used for email and text alarm alerts. </h3> <br>
+    {% if form.expert.errors %}
+      <h4> <font color='red'> {{ form.expert.errors[0] }} </font> </h4>
+    {% endif %}   
+    {% if form.phone.errors %}
+      <h4> <font color='red'> {{ form.phone.errors[0] }} </font> </h4>
+    {% endif %}   
+    {% if form.country.errors %}
+      <h4> <font color='red'> {{ form.country.errors[0] }} </font> </h4>
+    {% endif %}   
+    <form action="shifter_information" method="POST" role="form">
+      <table class="table table-hover">
+         <tr> 
+           <th> <h4> First name: </h4> </th>
+           <th> <h4> {{ form.firstname }} </h4> </th>
```

Comment:
You can render the whole field by importing the form helpers and then just doing:

```
{{ render_field(form.firstname) }}
```

This will correctly show the errors next to each item.